docs/release-notes/NuGet-5.10.md

```diff
@@ -0,0 +1,172 @@
+---
+title: NuGet 5.10 Release Notes
+description: Release notes for NuGet 5.10 including new features, bug fixes, and DCRs.
+author: zkat
+ms.author: kmarchan
+ms.date: 6/11/2021
+ms.topic: conceptual
+---
+
+# NuGet 5.10 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.10.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.10](https://visualstudio.microsoft.com/downloads/) | [5.0.300](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+  
+> [!NOTE]
+> Visual Studio 16.10, MSBuild 16.10, and .NET 5.0.300+ requires NuGet.exe 5.10 or later.
+
+## Summary: What's New in 5.10
+
+* Signing: implement dotnet trusted-signers command - [#8053](https://github.com/NuGet/Home/issues/8053)
+
+* Add the project type `nfproj` to the list of supportedProjectExtensions for Nuget CLI. - [#10562](https://github.com/NuGet/Home/issues/10562)
```

Comment:
I think we try to order by impact in the main section so this likely goes to the bottom.